File: Modul-2/Soal 5/PRAK205-2410817110009-MuhammadAlfiGunawan.py
import math
#case 1
tinggi,miring = map(float,input().split())

alas = math.sqrt((miring*miring)-(tinggi*tinggi))
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
keliling = alas + miring + tinggi
print(f"Keliling = {keliling:.0f} cm")
luas = 1.0/2.0 * alas * tinggi
print(f"Luas = {luas:.0f} cm^2")

#case 2
tinggi = float(input())
miring = float(input())

alas = math.sqrt((miring*miring)-(tinggi*tinggi))
print(f"Alas = {alas:.0f} cm")
print(f"Tinggi = {tinggi:.0f} cm")
keliling = alas + miring + tinggi
print(f"Keliling = {keliling:.0f} cm")
luas = 1.0/2.0 * alas * tinggi
print(f"Luas = {luas:.0f} cm^2")

File: Modul-2/Soal 5/PRAK205-2410817110009-MuhammadAlfiGunawan.c
#include<stdio.h>
#include<math.h>

int main(){
    float tinggi,miring,alas,keliling,luas;

    scanf("%f", &tinggi);
    scanf("%f", &miring);

    alas = sqrt((miring*miring)-(tinggi*tinggi));
    printf("Alas = %.0f cm\n", alas);
    printf("Tinggi = %.0f cm\n", tinggi);

    keliling = alas + miring + tinggi;
    printf("Keliling = %.0f cm\n", keliling);

    luas = 1.0/2.0 * alas * tinggi;
    printf("Luas = %.0f cm^2", luas); 

}